src/accelerate/commands/config/cluster.py

```diff
@@ -99,6 +99,7 @@ def get_cluster_input():
                 default=1,
             )
 
+    fsdp_config = None
```

Comment:
```suggestion
    fsdp_config = {}
```
Can you remove the same thing at line 112 as a result?